### PR TITLE
fix breadcrumbs; add auth guard

### PR DESCRIPTION
**Pull request by alena-khmaruk**

### CODE REVIEW

File: src/app/pages/course-item-page/course-form/course-form.component.html [hunk 1]
```diff
@@ -1,12 +1,28 @@
 <div class="vc-form">
-    <h3 class="vc-form__name">New Course</h3>
-    <form id="course" class="vc-form__block" name="course">
+    <h3 class="vc-form__name" *ngIf="!isEditPage">New Course</h3>
+    <h3 class="vc-form__name" *ngIf="isEditPage">Edit {{course.title}} Course</h3>
```
Comment (LenkouAndrei) [inline src/app/pages/course-item-page/course-form/course-form.component.html]: can you, please create property headline, define it in OnInit and use here as <h3 class="vc-form__name">{{ headline }}</h3>

File: src/app/shared/components/header/services/authentication.service.ts [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
 
 @Injectable({
     providedIn: 'root'
 })
 export class AuthenticationService {
-    private _user: ReplaySubject<User> = new ReplaySubject<User>();
+    private _user: BehaviorSubject<User> = new BehaviorSubject<User>(null);
+
+    constructor() {
+        const token: string = localStorage.getItem('token');
```
Comment (LenkouAndrei) [inline src/app/shared/components/header/services/authentication.service.ts]: move 'token' to prop of AuthService or const you use it in different methods

File: src/app/pages/course-item-page/course-form/course-form.component.html (hunk 1 above)
Comment (alena-khmaruk) [reply src/app/pages/course-item-page/course-form/course-form.component.html]: Done

File: src/app/shared/components/header/services/authentication.service.ts (hunk 2 above)
Comment (alena-khmaruk) [reply src/app/shared/components/header/services/authentication.service.ts]: Done

**Comment by alena-khmaruk** [comment]
@LenkouAndrei comments were resolved. Please review it again.

**Pull request merged**